components/lcd_touch/esp_lcd_touch_gt911/include/esp_lcd_touch_gt911.h

```diff
@@ -41,6 +41,15 @@ esp_err_t esp_lcd_touch_new_i2c_gt911(const esp_lcd_panel_io_handle_t io, const
 #define ESP_LCD_TOUCH_IO_I2C_GT911_ADDRESS          (0x5D)
 #define ESP_LCD_TOUCH_IO_I2C_GT911_ADDRESS_BACKUP   (0x14)
 
+/**
+ * @brief GT911 Configuration Type
+ *
+ */
+typedef struct {
+    uint32_t dev_addr;  /*!< I2C device address */
+    void *user_data;    /*!< User data passed to callback */
```

Comment:
Where is it used?

Reply:
`dev_addr` : where the user needs to place the [address](https://github.com/espressif2022/esp-bsp/blob/e20fa5e8b497fffe6172787de0ff59d7e2ebbc10/components/lcd_touch/esp_lcd_touch_gt911/esp_lcd_touch_gt911.c#L120) .
`user_data`:I asked zhongwei, his `arduino` lib needs to use the `user_data` field, so I still reserved this field for him.

Reply:
I think, that this is not a good solution. I am thinking about it and it will be better to add new item `void *driver_data;` into `esp_lcd_touch_config_t` structure. And left `user_data` free.

Reply:
Yes, I agree. Independent `driver_data` is clearer and more compatible.